### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by prodion23**
Drops 1.18 support

### CODE REVIEW

File: examples/grpc-client/main.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:build ignore
```
Comment (tim-mwangi) [inline examples/grpc-client/main.go:1]: oh these are now needed?

Comment (prodion23) [reply examples/grpc-client/main.go:1]: Hmm,I think these were added with fmt, but we probably don't need both lines 1 & 2

Comment (tim-mwangi) [reply examples/grpc-client/main.go:1]: cool. let's remove them.

**Review by tim-mwangi — APPROVED** [review]

**Comment by tim-mwangi** [comment]
Hey @prodion23 can we file JIRA to look at some of these gosec issues? https://github.com/hypertrace/goagent/pull/214/checks?check_run_id=14688939422 Thanks.

**Pull request merged**